src/snapshot-editor/src/info.rs

```diff
@@ -87,3 +94,17 @@ fn info_vcpu_states(state: &MicrovmState, _: u16) -> Result<(), InfoVmStateError
     }
```

Comment:
If x86 now has vcpu states printable, we should make `info_vcpu_states` be available on x86 as well.

Reply:
done.